src/Service.Tests/GraphQLBuilder/Sql/SchemaConverterTests.cs

```diff
@@ -741,7 +741,7 @@ public static Entity GenerateEmptyEntity(string entityName)
         {
             return new Entity(
                 Source: new($"{SCHEMA_NAME}.{TABLE_NAME}", EntitySourceType.Table, null, null),
-                Rest: new(EntityRestOptions.DEFAULT_SUPPORTED_VERBS),
+                Rest: new(new SupportedHttpVerb[] { }),
```

Comment:
empty?

Reply:
Yeah, for table, rest methods should ideally be empty because they are meant for stored procedures.

Reply:
We don't allow anything other than string as a value for `rest.path`. Any other datatype would fail deserialization.